include/inviwo/qt/editor/processorgraphicsitem.h

```diff
@@ -119,18 +124,23 @@ class IVW_QTEDITOR_API ProcessorGraphicsItem : public EditorGraphicsItem,
     virtual void onProcessorReadyChanged(Processor*) override;
     virtual void onProcessorPortAdded(Processor*, Port*) override;
     virtual void onProcessorPortRemoved(Processor*, Port*) override;
-#if IVW_PROFILING
     virtual void onProcessorAboutToProcess(Processor*) override;
     virtual void onProcessorFinishedProcess(Processor*) override;
-#endif
+
+    virtual void onProcessorStartBackgroundWork(Processor*, size_t jobs) override {
+        backgroundJobs_ += jobs;
+    };
+    virtual void onProcessorFinishBackgroundWork(Processor*, size_t jobs) override {
+        backgroundJobs_ -= jobs;
```

Comment:
overflow/underflow handling due to size_t?

Reply:
As long as nothing else is wrong, that should never happen...